AlphaWallet/Tokens/Collectibles/Views/TokenHistoryChartView.swift

```diff
@@ -50,6 +50,11 @@ class TokenHistoryChartView: UIView {
         chartView.rightAxis.setLabelCount(5, force: true)
         chartView.rightAxis.valueFormatter = valueFormatter
 
+        let marker = XYMarkerView(
```

Comment:
This seem to shadow `marker` with a local variable with a duplicate definition?

Reply:
Issue was filed in #6279

Reply:
Huh? Then you should have removed the `marker` properly since you moved it to a local variable in this PR. Instead of making an unused one. On the other hand, it shouldn't have been included in this PR since it's about colors.

Reply:
Fixed in #6287